tests/colr_to_svg.py

```diff
@@ -23,6 +23,12 @@
 from typing import Tuple
 from fontTools.pens import svgPathPen
 from fontTools.pens import transformPen
+from fontTools.ttLib.tables import otTables
+
+
+_PAINT_SOLID = 1
```

Comment:
we could even make this an IntEnum inside nanoemoji.paint module and use if for the various Paint*.Format class attributes